cairo/tests/ethereum/cancun/test_fork.py

```diff
@@ -201,22 +218,25 @@ def test_make_receipt(
             "make_receipt", tx, error, cumulative_gas_used, logs
         )
 
-    @given(env=..., tx=tx_without_code())
-    @settings(max_examples=100)
+    @given(env=env_with_valid_gas_price(), tx=tx_without_code())
     def test_process_transaction(self, cairo_run, env: Environment, tx: Transaction):
         try:
-            gas_used_cairo, logs_cairo, error_cairo = cairo_run(
-                "process_transaction", env, tx
-            )
+            env_cairo, gas_cairo, logs_cairo = cairo_run("process_transaction", env, tx)
         except Exception as e:
             with strict_raises(type(e)):
-                process_transaction(env, tx)
+                output_py = process_transaction(env, tx)
+                # The Cairo Runner will raise if an exception is in the return values.
+                if len(output_py) == 3:
+                    raise output_py[2]
+                assert env_cairo == env
+                assert gas_cairo == output_py[0]
+                assert logs_cairo == output_py[1]
```

Comment:
if proces_transaction raises, the following lines are not executed, you need to put them out of the context manager

Reply:
I think I just need to put them right above the raise output-py[2]?